enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.stories.tsx

```diff
@@ -35,6 +35,34 @@ export default {
       control: { type: "radio" },
     },
     questionId: questionIdArgType,
+    entityTypes: {
```

Comment:
This adds a control in storybook for EZ testing. Visit http://localhost:6006/?path=/story/embeddingsdk-interactivequestion--create-question to test it out.
![image](https://github.com/user-attachments/assets/b0a241d2-71e8-4b34-9c44-ebf2ad33262c)